extensions/amp-analytics/amp-analytics.md

```diff
@@ -1374,6 +1374,12 @@ AMP component developers can implement collection of data using AMP analytics.
 For more information, please refer to
 [Implementing analytics for AMP components](amp-components-analytics.md).
 
+### Google Analytics 4 and AMP
+
+For information on how to set up Google Analytics 4 with amp-analytics see
+https://developers.google.com/analytics/devguides/collection/amp-analytics and
```

Comment:
Seems like this link is still talking about measurement_id and links to https://support.google.com/analytics/answer/1008080#trackingID&zippy=%2Cin-this-article, where it shows the process of setting up UA instead of GA4. 

Is this link going to be updated soon by GA folks? So maybe it is just out-of-sync for now?

Reply:
thanks for flagging, i believe that is the case. ill double check with them